### PR TITLE
Add nn.padding to docs fixes #1127

### LINKED ISSUE #1127: Add all padding modules to the docs
Is there a way to use reflection padding instead of zero padding with nn.Conv2d ? I see there is a [legacy implementation](https://github.com/pytorch/pytorch/blob/master/torch/legacy/nn/SpatialReplicationPadding.py) but the option is not in nn.Conv2d.
Thanks,

### PR DESCRIPTION
**Pull request by nrbrd**
This PR adds all nn.padding modules to the docs. #1127  
* ReflectionPad2d
* ReplicationPad2d
* ReplicationPad3d
* ZeroPad2d
* ConstantPad2d

### CODE REVIEW

**Comment by soumith** [comment]
thanks!
